addons/project/models/project.py

```diff
@@ -1143,7 +1143,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         ('october', 'October'),
         ('november', 'November'),
         ('december', 'December'),
-    ], compute='_compute_repeat', readonly=False)
+    ], compute='_compute_repeat', readonly=False, compute_sudo=True)
 
     repeat_show_dow = fields.Boolean(compute='_compute_repeat_visibility')
     repeat_show_day = fields.Boolean(compute='_compute_repeat_visibility')
```

Comment:
Those fields (using the `_compute_repeat_visibility`) should also have the group, I think. 🙂